### DISCUSSION TITLE
Dev Diary #7: Black Temple, Mythics and more!

### DISCUSSION THREAD

**Discussion by Nyeriah** [chromiecraft/chromiecraft#7329] [General]
Greetings, time-travelers!

It's been a long time coming, but... our latest dev diary is finally here!

As always, we’re excited to share updates on what we’ve been working on and what’s on the horizon. There’s a lot to cover, so let’s dive right in!

**The Black Temple, not Bug Temple**

It’s been a challenging year for us all, as you might know from our recent announcements. Combined with real-life struggles—work, family, and the usual ups and downs—our ability to deliver content at the quality we aim for has been impacted. Unfortunately, this led to the issues with Battle of Mount Hyjal.

We acknowledge that Hyjal didn't meet the standards we strive for, and we understand it was disappointing for many of you. As a result, we’re re-evaluating our entire progression process to prevent repeating past mistakes. This means an extended testing and development phase to ensure everything is up to scratch.

With the Black Temple, we’re committed to reaching that golden standard once again. We plan to release it only when we’ve ironed out all major bugs and avoid the need for significant post-release fixes. This is why we haven’t set a specific release date for the Betrayer just yet, though we might be very close.

Your feedback has been invaluable. If you encounter issues during testing, please report them. You might even earn double points for your efforts!

**Black Temple: Status Report**

On the development front, we’ve addressed most of the reported issues. We are aware that as more people get involved with the testing process, more issues will be reported, and we are eager to address them so we can speed up the release. As it stands, few issues remain, and we’re also working on an overhaul for Illidan Stormrage, which should be ready in the next couple of weeks.

We’ve applied the first set of tuning changes to the PTR. Check it out and let us know your thoughts and suggestions.

In summary, we’re focusing on resolving the last remaining issues and fine-tuning the raid. That’s all that’s left to do!

**Tuning Transparency Efforts**

Many of you have asked for greater transparency in tuning values, and we’ve listened. You can now view all current changes here: https://github.com/chromiecraft/PTR-review/tree/master/current_raid_tuning

This repository contains the database values we’re using. We may move this information to a different module in the future, but we’ll keep you updated if that happens.

**The Return of Mythics**

The Mythic difficulty mode is making a return with the Black Temple! It will follow the same rules and structure as in previous raids like Magtheridon's Lair and Gruul’s Lair (it will take a similar amount of time to acquire items, but sorry hunters, Warglaives of Azzinoth will not be available at the Mythic reward exchanger).

We’re not adding support for previous raids at this time, but it’s something we might consider in the future as we continue updating our scripts.

We hope this brings an enjoyable experience for all, whether you’re looking for an easy run or a challenging journey.

**And... What About the Survey?**

We’re still working on implementing changes from our latest player survey, including increased XP/gold for low-level RDF and more badges in raids. Expect an update on these changes as soon as next week. Stay tuned!

It’s Happy Hour… Somewhere! Brewfest Updates

Core Direbrew will be dropping level 70 items this year and will also receive a stat boost to make him more challenging. We’ve got some cool surprises in store, so keep an eye out for more updates!

**Chromie Needs You!**

We’ve achieved incredible things with the help of our contributors, but there’s still more to be done. If you’re interested in helping us make Chromiecraft a better place for everyone, check out our latest announcements on Discord. There’s bound to be a role that’s perfect for you. Don’t be shy!

That’s all for now. We’ve got more to share, but we’ll save that for next time. Thanks for your time and happy hunting!

Regards

**Comment by rakrak1775** [comment]
good job love u

**Comment by armchairrockstar1** [comment]
I don't like Mythic's at all. All that extra effort for collecting points...

**Comment by assassinbait** [comment]
BT and SWP mythic item cost can't be the same cost as mag and gruul. BT has 9 bosses and it'd take 2 raids only to get 1 item

otherwise pog

**Comment by Shardaz** [comment]
Thanks for the hard work you put on bt so far! 

Thoughts about mythic raids:

If that is what the majority of community wants, its fine for me, although I personally hate it.

 

BUT: 10 Mythic points to buy an item is just insane. This means 22.5 extra items of choice per raid!

This will kill the whole bracket after like 5 weeks. Everyone will have everything they need and want.

Also this will make joining a mythic raid basically mandatory, because of the big advantage it has. Which will again destroy the idea behind it, as everyone will run the mythic version.

 

This also takes away the joy of finally getting a rare and nice item! This is completely against the blizzard design philosophy and why people like the game.

Like this it is just a funserver with extra steps. You can just do the first few easy bosses a couple weeks and buy whatever item you want.

 

Please don’t listen to players that have no idea about good game design. Leave that to the (original) blizzard developers, that made the game like we enjoy it and how we signed up for it by joining a blizzlike server (Yes, it was still labelled as this when a lot of the old playerbase joined).

 

My idea: Spawn a chest after defeating the whole mythic dungeon, that contains like 3-5 random extra pieces of loot.

If you really want to stick to the point system, make the prices like 50-100 points, depending on item category. Just as a bad luck protection.

**Comment by Nyeriah** [comment]
We understand the concerns raised. This point needs clarification and we do apologize for it, as where it reads "the costs will be the same" should be read as "it will take a similar amount of time to acquire items", considering, as mentioned, BT has way more bosses so you will be getting way more points per week. Rectified.

**Reply by Shardaz** [reply]
Thanks for clarification. Imo, you should still consider putting the powerful stuff like  trinkets, weapons etc. at a much higher price

**Reply by shaszun** [reply]
> We understand the concerns raised. This point needs clarification and we do apologize for it, as where it reads "the costs will be the same" should be read as "it will take a similar amount of time to acquire items", considering, as mentioned, BT has way more bosses so you will be getting way more points per week. Rectified.

Keep in mind most people are already almost full BIS from Hyjal. This means they will just need probably up to 3 tokens from BT and like 2-3 side pieces. So yea, if it takes like 1 month to get a side armor piece, then a weapon or more rare armor should be at 1.5-2 months of full BT raiding. Main concern is that the raid can be dead in 1 month of release if gear is easy to acquire. Ilidan pieces might even take 3 months of raiding in mythic to buy. Some are extremely good items. Might also consider adding Hyjal items to the vendor, loot pool tables are not working correct in my opinion. Our guild is extremely unlucky with the drops, we got the same loot from some of the bosses for like 4-5 weeks in a row. There were some items that only once dropped so far, there are some pieces that I haven't see yet dropping too. Statisticaly this is extremely low chance to happen. Imagine our frustration that we had to even to change the person creating the raid out of superstition, because it was the same person all that time...

**Comment by armchairrockstar1** [comment]
This mythic thing just goes against the philosophy of the game. Please DON'T bring it back!
